packages/genesys-spark-components/src/components/beta/gux-calendar-beta/components/single/gux-calendar.scss

```diff
@@ -9,7 +9,7 @@
   background: var(--gse-ui-calendarMenu-date-selected-foregroundColor);
   border-radius: var(--gse-ui-calendarMenu-single-body-borderRadius);
   // TODO: Get box-shadow token working instead of hardcoding
-  // box-shadow: var(--gse-ui-calendarMenu-boxShadow-color)px var(--gse-ui-calendarMenu-boxShadow-x)px var(--gse-ui-calendarMenu-boxShadow-y)px var(--gse-ui-calendarMenu-boxShadow-blur)px var(----gse-ui-calendarMenu-boxShadow-spread)px;
+  // box-shadow: var(--gse-ui-calendarMenu-boxShadow-color)px var(--gse-ui-calendarMenu-boxShadow-x, --gse-ui-calendarMenu-boxShadow-offsetX)px var(--gse-ui-calendarMenu-boxShadow-y, --gse-ui-calendarMenu-boxShadow-offsetY)px var(--gse-ui-calendarMenu-boxShadow-blur)px var(----gse-ui-calendarMenu-boxShadow-spread)px;
```

Comment:
bit confused on this change. accidental find and replace?

Reply:
wait  you can specify multiple variables in one var()?

Reply:
@jordanstith15 2nd one is the fallback.